drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -204,7 +183,7 @@ def do_attempt(self, attempt, suffix=''):
                 # If this item has no input value set, we are done with it.
                 is_correct = True
                 feedback = item['feedback']['correct']
-            state = {'top': attempt['top'], 'left': attempt['left']}
+            state = {'top': attempt['top'], 'left': attempt['left'], 'position': 'absolute'}
```

Comment:
Would it be possible to avoid adding more CSS in the JS code in this case? I understand the need to specify the coordinates in the the code, but can `position: absolute` be set in the CSS files? Or via a class if this needs to change dynamically?